Author: Benoit Mortier
Description: put the Debian directory for prototype into the script tag

--- a/core/ihtml/themes/default/password.tpl	2013-11-16 18:49:49.000000000 +0100
+++ b/core/ihtml/themes/default/password.tpl	2013-11-16 18:50:53.565148013 +0100
@@ -12,7 +12,7 @@
   <style type="text/css">@import url('themes/default/password-style.css');</style>
   <link rel="shortcut icon" href="favicon.ico">
 
-  <script src="include/prototype.js" type="text/javascript"></script>
+  <script src="/javascript/prototype/prototype.js" type="text/javascript"></script>
   <script src="include/fusiondirectory.js" type="text/javascript"></script>
   <script src="include/pwdStrength.js" type="text/javascript"></script>
 </head>
